.github/workflows/prerelease.yml

```diff
@@ -156,7 +163,16 @@ jobs:
         shell: pwsh
         run: |
           build\windows\package_msi.ps1 -integration "$env:INTEGRATION" -arch ${{ matrix.goarch }} -tag "$env:TAG" -pfx_passphrase "$env:PFX_PASSPHRASE" -pfx_certificate_description "$env:PFX_CERTIFICATE_DESCRIPTION"
+      - name: Test win packages installation
+        uses: paologallinaharbur/test-packages-action/windows@add_win_test
```

Comment:
This needs to be changed to the final repository/version before merge

Reply:
Agreed. I think the opensource group already has a repo for actions, at least form what I can tell from the repolinter action, they use: **newrelic/repolinter-action@v1**